sklearn/utils/estimator_checks.py

```diff
@@ -597,7 +598,7 @@ def check_fit2d_1sample(name, estimator_orig):
 
 @ignore_warnings
 def check_fit2d_1feature(name, estimator_orig):
-    # check by fitting a 2d array and prediting with a 1d array
+    # check by fitting a 2d array with only one feature
     rnd = np.random.RandomState(0)
     X = 3 * rnd.uniform(size=(10, 1))
     y = X[:, 0].astype(np.int)
```

Comment:
Fitting a 2d array with only 1 feature is either OK or raise a ValueError depending on the estimator. Should we remove this common check then?

We might want to remove `check_fit2d_1sample` as well which is either OK or raise a ValueError depending on the estimator.